Assets/PatchKit Patcher/Scripts/PatcherStatistics.cs

```diff
@@ -47,6 +71,30 @@ public static IEnumerator SendEvent(string eventName, string appSecret)
             json["operating_system_family"] = operatingSystemFamily;
             json["operating_system_version"] = operatingSystemVersion;
 
+            if (parameters.HasValue)
+            {
+                var v = parameters.Value;
+                if (v.VersionId.HasValue)
+                {
+                    json["version_id"] = v.VersionId.Value;
```

Comment:
Please refactor "version_id" and similar parameters to constants.

Reply:
I agree with the enums for statistic events because that allows us to check for errors more quickly and helps avoid situations where somebody would supply an event_name with a typo, or something like that.

In this particular situation, the "version_id" and so on are only used once and only in this context. Refactoring them to constants would yield no significant benefit.

Reply:
It can be an enum.

> Refactoring them to constants would yield no significant benefit.

It's an aesthetic matter. If someone would re-use "version_id" in the future for some particular reason, he/she is more likely to write down "version_id" string again instead of refactoring your usage to a constant.